src/uu/rm/src/rm.rs

```diff
@@ -255,7 +255,18 @@ fn handle_dir(path: &Path, options: &Options) -> bool {
             // correctly on Windows
             if let Err(e) = remove_dir_all(path) {
                 had_err = true;
-                show_error!("could not remove '{}': {}", path.display(), e);
+                if e.kind() == std::io::ErrorKind::PermissionDenied {
+                    // GNU compatibility (rm/fail-eacces.sh)
```

Comment:
I understood this comment first as saying that you are trying to match GNU. I only realized that you were intentionally breaking full compatibility when I read the changes in `build-gnu.sh`. Maybe you could make that a little bit clearer?

Reply:
sorry, i mixed changes
1. I reached full compatibility on this test rm/fail-eacces.sh
2. for other, it will be too complex, so, i am updating the GNU test to match what we have

is that clear?

Reply:
Alright! Thanks!